js-miniapp-sample/src/pages/tests/user-details.test.js

```diff
@@ -26,4 +28,14 @@ describe('Test Reducer', () => {
     const newState = dataFetchReducer(initialState, { type: 'FETCH_FAILURE' });
     expect(newState.isError).toBe(true);
   });
+
+  test('The points init action updates state properly', () => {
+    const newState = dataFetchReducer(initialState, { type: 'POINTS_FETCH_INIT' });
+    expect(newState.isLoading).toBe(true);
+  });
+
+  test('The points failure action updates state properly', () => {
+    const newState = dataFetchReducer(initialState, { type: 'POINTS_FETCH_FAILURE' });
+    expect(newState.isError).toBe(true);
+  });
```

Comment:
We don't usually add unit tests for the JS Sample App because it's not used in production, so having tests can add unnecessary overhead.

Actually we should probably just remove all of the sample app tests sometime 🤔

Reply:
I agree on this @corycaywood

Reply:
ok I've removed the two tests 😄